analysis/paper/paper.qmd

```diff
@@ -203,20 +182,15 @@ Word-level tokenisation was applied with the `unnest_tokens()` function of the `
 ```{r}
 #| label: stop-words
 
-stop_words_custom <- emu_theses %>% 
-  unnest_tokens(output = word, input = title) %>%   ### add words from the title
-  select(word) %>% 
-  rbind(., data.frame(word = c("preface", "foreword", "introduction", "conclusion", "thesis", 
-                               "source", "author"))) %>% 
-  # rbind(., data.frame(word = c("city", "urban", "urbanism"))) %>% 
-  rbind(., data.frame(word = c("hab", "km")))
+emu_theses$text_clean <- emu_theses$text_clean |>
+  str_remove_all(find_meta_stopwords(emu_theses)) |> 
+  str_remove_all(subject_stopwords()) |> 
+  str_remove_all(document_stopwords())
```

Comment:
I moved these lines from the `clean-text` code chunk to the `stop-words` code chunk. 

This removes:
1. all metadata stopwords on the document level and
2. all subject-specific and document stopwords on the level of the corpus

These are stop phrases, to be precise, so it is better to remove them before tokenizing the text.